compaction.go

```diff
@@ -416,8 +423,10 @@ func (c *compaction) makeInfo(jobID int) CompactionInfo {
 	}
 	if c.outputLevel != nil {
 		info.Output.Level = c.outputLevel.level
+
 		// If there are no inputs from the output level (eg, a move
 		// compaction), add an empty LevelInfo to info.Input.
+		// why not check c.kind?
```

Comment:
small piece of code i didn't quite understand that I modified. Why not check c.kind here?